tap_recharge/client.py

```diff
@@ -190,6 +190,12 @@ def request(self, method, path=None, url=None, **kwargs): # pylint: disable=too-
         if path == 'collections':
```

Comment:
How this PR is different then https://github.com/singer-io/tap-recharge/pull/32/files
Can we merge everything in one?

Reply:
This, PR contains all the changes related to the Products stream. And PR #32 is general missing fields PR for all the streams.